roles/support_prometheus/tasks/main.yml

```diff
@@ -0,0 +1,30 @@
+- name: Write new entry to prom-cosmos.yml
+  blockinfile:
+    path: "/etc/prometheus/{{ cosmos_prom_file }}.yml"
```

Comment:
where is cosmos_prom_file defined? Add it as a default for the playbook role. <role_dir>/defaults/main.yml.

To create defaults, simply add a defaults/main. yml file in your role directory. These variables will have the lowest priority of any variables available, and can be easily overridden by any other variable, including inventory variables.